src/test/claimtriebranching_tests.cpp

```diff
@@ -844,15 +844,28 @@ BOOST_AUTO_TEST_CASE(get_claim_by_id_test)
     fixture.MakeSupport(fixture.GetCoinbase(), tx1, name, 4);
     fixture.IncrementBlocks(1);
 
-    CMutableTransaction tx2 = fixture.MakeClaim(fixture.GetCoinbase(), name, "two", 1);
+    CMutableTransaction tx2 = fixture.MakeClaim(fixture.GetCoinbase(), name, "two", 2);
```

Comment:
Why is this changing?

Reply:
typo, fixing. I meant to do this only to tx1